packages/manhattan/src/index.js

```diff
@@ -91,83 +93,76 @@ const ManhattanPlot = ({
     </text>
   )
 
-  const Yticks = () => {
-    return (
-      <g>
-        {yScale.ticks().map((t) => {
-          return (
-            <g key={t}>
-              <line
-                x1={padding}
-                x2={width}
-                y1={yScale(t)}
-                y2={yScale(t)}
-                stroke={'#BDBDBD'}
-              />
-              <text
-                className={'yTickText'}
-                style={{ textAnchor: 'middle' }}
-                x={padding - 15}
-                y={yScale(t) + 5}
-              >
-                {t}
-              </text>
-            </g>
-          )
-        })}
-      </g>
-    )
-  }
+  const yAxisTicks = (
+    <g>
+      {yScale.ticks().map((t) => {
+        return (
+          <g key={t}>
+            <line
+              x1={padding}
+              x2={width}
+              y1={yScale(t)}
+              y2={yScale(t)}
+              stroke={'#BDBDBD'}
+            />
+            <text
+              className={'yTickText'}
+              textAnchor={'middle'}
+              x={padding - 15}
+              y={yScale(t) + 5}
+            >
+              {t}
+            </text>
+          </g>
+        )
+      })}
+    </g>
+  )
 
-  const Xlabel = () => (
+  const xAxisLabel = (
     <text
       className={'xLabel'}
-      style={{ textAnchor: 'middle' }}
+      textAnchor={'middle'}
       x={width / 2}
       y={height - (padding / 4)}
     >
       Chromosome
     </text>
   )
 
-  const Xticks = () => {
-    const snpsSplitByChrom = plotChromosomes.map(chr =>
-      ({
-        name: chr,
-        data: data.filter(snp => chr === `chr${snp.chromosome}`),
```

Comment:
Previously, this `filter` inside of `map` required iterating over the `data` array once per chromosome (22-24 times). This change reduces it to one iteration.